Twt shell cmd doc update (#11365)
nrfconnect/sdk-nrf: samples/wifi/shell/README.rst

   * - Subcommands
     - Description
   * - scan
     - Scan for Wi-Fi APs
   * - connect
     - | Connect to Wi-Fi AP with below parameters:
       | <SSID>
       | <Channel number> (optional: 0 means all)
       | <PSK> (optional: valid only for secured SSIDs)
       | <Security type> (optional: 0-None, 1-PSK, 2-PSK-256, 3-SAE)
       | <MFP> (optional: 0-Disable, 1-Optional, 2-Required)
   * - disconnect
     - Disconnect from the Wi-Fi AP
   * - status
     - Status of the Wi-Fi interface
   * - statistics
     - Wi-Fi interface statistics
   * - ap
     - | Access Point mode commands

>        | enable - Configure the Wi-Fi interface as access point mode

Reviewer: Enable Access Point mode, with the following parameters: 
<List parameters from the shell>

>        | disable - Configure the Wi-Fi interface as station mode

Reviewer: Match description to the one from the shell

>        | Currently access point mode is not supported

Reviewer: This should be a note, i.e. Note : Access point mode is presently unsupported.

   * - Subcommands
     - Description
   * - scan
     - Scan for Wi-Fi APs
   * - connect

>      - | Connect to Wi-Fi AP with below parameters:

Reviewer 2: ```suggestion
     - | Connect to the Wi-Fi AP with the following parameters:
```

       | <SSID>
       | <Channel number> (optional: 0 means all)
       | <PSK> (optional: valid only for secured SSIDs)
       | <Security type> (optional: 0-None, 1-PSK, 2-PSK-256, 3-SAE)
       | <MFP> (optional: 0-Disable, 1-Optional, 2-Required)
   * - disconnect
     - Disconnect from the Wi-Fi AP
   * - status
     - Status of the Wi-Fi interface
   * - statistics
     - Wi-Fi interface statistics
   * - ap
     - | Access Point mode commands

>        | enable - Enable Access Point mode with below parameters

Reviewer 2: ```suggestion
       | enable - Enable Access Point mode, with the following parameters:
```

       | <SSID>
       | <SSID length>
       | channel>
       | <psk> [optional]
       | disable - Disable Access Point mode

>        | (Note: Access point mode is presently not supported.)

Reviewer 2: ```suggestion
       | (Note that the Access Point mode is presently not supported.)
```

       | setup - Start a TWT flow:
       | <negotiation_type: 0 - Individual, 1 - Broadcast, 2 - Wake TBTT>
       | <setup_cmd: 0 - Request, 1 - Suggest, 2 - Demand>

>        | <dialog_token: 1-255> <flow_id: 0-7> <responder: 0/1> <trigger: 0/1>

Reviewer 2: ```suggestion
       | <dialog_token: 1 - 255> <flow_id: 0 - 7> <responder: 0/1> <trigger: 0/1>
```

>        | <implicit: 0/1> <announce: 0/1> <twt_wake_interval: 1µs-262144µs>
>        | <twt_interval: 1µs-2^31µs>

Reviewer 2: ```suggestion
       | <implicit: 0/1> <announce: 0/1> <twt_wake_interval: 1 µs to 262144 µs>
       | <twt_interval: 1 µs to 2^31 µs>
```

       |
       | teardown - Teardown a TWT flow:
       | <negotiation_type: 0 - Individual, 1 - Broadcast, 2 - Wake TBTT>
       | <setup_cmd: 0 - Request, 1 - Suggest, 2 - Demand>

>        | <dialog_token: 1-255> <flow_id: 0-7>

Reviewer 2: ```suggestion
       | <dialog_token: 1 - 255> <flow_id: 0 - 7>
```
